xbmc/interfaces/legacy/Control.cpp

```diff
@@ -261,15 +261,17 @@ namespace XBMCAddon
 
     String ControlButton::getLabel()
     {
-      if (!pGUIControl) return NULL;
+      if (!pGUIControl)
```

Comment:
Bonus points if you do `if (pGUIControl != nullptr)`. This contextualizes the comparison to be a comparison of pointers, whereas currently with `!` we suggest that the variable has a boolean context.

Reply:
sure, done